<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDialog</class>
 <widget class="QDialog" name="TDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QPushButton{
	font: 10pt &quot;华文中宋&quot;;
}</string>
  </property>
  <widget class="QLabel" name="nowTimeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>176</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="timeStartButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:rgba(231,123,34,90)</string>
   </property>
   <property name="text">
    <string>开始定时</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeTimeButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:rgba(231,123,34,90)</string>
   </property>
   <property name="text">
    <string>关闭定时</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exutTimeButton">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>8</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,0) url(:/image/icon/cross_15.977464788732px_1153732_easyicon.net.png) no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="timeBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="theFinalLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
